### PR TITLE
Release 4.0.2 -- Hide debug detail

### PR DESCRIPTION
**Pull request by briskt**

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -63,16 +64,17 @@ func init() {
...
 	if debugStr == "TRUE" || debugStr == "true" {
 		lib.EnableDebug()
+		debugMode = true
 	}
 
 	if readOnlyMode {
 		lib.EnableReadOnlyMode()
 	}
+
+	getToken()
```
Comment (hobbitronics) [inline cmd/root.go:77]: strange this wasn't called before. Why the addition now?

File: .github/workflows/changelog.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        ref: main
+    - uses: rhysd/changelog-from-release/action@v3
+      with:
+        file: CHANGELOG.md
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        pull_request: true
```
Comment (hobbitronics) [inline .github/workflows/changelog.yml:18]: Nice. Does this need to be mentioned in documentation?

**Review by hobbitronics — APPROVED** [review]

File: .github/workflows/changelog.yml (hunk 2 above)
Comment (briskt) [reply .github/workflows/changelog.yml:18]: Maybe. It's sort of just an experiment at this point, though.

File: cmd/root.go (hunk 1 above)
Comment (briskt) [reply cmd/root.go:77]: I just moved it lower so that it runs after `debugMode` is set.

**Pull request merged**